toolbox

```diff
@@ -769,6 +772,20 @@ create()
 	home_link="--home-link"
     fi
 
+    if [ -z "$prefix_sudo" ]; then
+	dnf_cache_dir="$HOME/.cache/toolbox/$fgc"
```

Comment:
I must be missing something here, so please bear with me. How does `~/.cache/toolbox/$fgc` get populated on the host?